modules/core/src/main/java/org/apache/ignite/internal/client/GridClientClusterState.java

```diff
@@ -47,7 +47,18 @@
      * Changes cluster state to {@code newState}.
      *
      * @param newState New cluster state.
+     * @param forceDeactivation If {@code true}, cluster deactivation will be forced.
      * @throws GridClientException If the request to change the cluster state failed.
+     * @see ClusterState#INACTIVE
+     */
+    public void state(ClusterState newState, boolean forceDeactivation) throws GridClientException;
```

Comment:
This API looks odd to me. Let's assume I call the method with the following parameters: `state(ClasterStae.ACTIVE, true);`
Does it mean that cluster will be forcibly deactivated in any way? Maybe `forceDeactivation` will be just ignored in this case. 
Long story short, it would be nice to update javadoc, at least it makes sense to mention that `forceDeactivation` is used only for cases when new cluster state is `INACTIVE`.

Reply:
but, existing java doc already highlight this? i hope : _If {@code true}, cluster deactivation will be forced._

Reply:
i extend a bit existing java doc

Reply:
I think that this "extended" javadoc should be applied to all places.
For example, `GridClientConnection#changeState(ClusterState state, UUID destNodeId, boolean forceDeactivation)`